docs/tutorial/index.md

```diff
@@ -1,609 +1,553 @@
-                                                                version
-----------------------------------------------------------------------------------------------------------------------------------------
- PostgreSQL 14.10 (Ubuntu 14.10-0ubuntu0.22.04.1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0, 64-bit
-(1 row)
+```{terminal}
+:user: ubuntu
+:host: my-juju-vm
+:input: juju bootstrap microk8s my-first-juju-controller
+Creating Juju controller "my-first-juju-controller" on microk8s/localhost
+Bootstrap to Kubernetes cluster identified as microk8s/localhost
+Creating k8s resources for controller "controller-my-first-juju-controller"
+Downloading images
+Starting controller pod
+Bootstrap agent now started
+Contacting Juju controller at 10.152.183.180 to verify accessibility...
 
-```
+Bootstrap complete, controller "my-first-juju-controller" is now available in namespace "controller-my-first-juju-controller"
 
-Type `exit` to get back to your unit shell and then again to return to your Multipass VM shell.
+Now you can run
+	juju add-model <model-name>
+to create a new model to deploy k8s workloads.
 
+```
 
+This will use ingredients from your client (especially the Juju agent binaries), the `juju-controller` charm from Charmhub and a pod from MicroK8s (backed by your current node -- your VM) to give you a running Juju controller.
 
 
+Take a moment to appreciate what this looks like:
+- On the MicroK8s side (in MicroK8s' database) you've now got:
+  1. a 'controller' namespace (`kubectl get namespaces`) holding
+  1. a 'controller-0' pod (`kubectl -n controller-my-first-juju-controller get pods`) holding
+  1. 3 containers: 'api-server', 'charm', 'mongodb' (`kubectl -n controller describe pod controller-0`).
+- On the Juju side (in the Juju controller's database) you've now got:
+  1. a Juju 'my-first-juju-controller' controller (i.e., control plane), with access to the 'microk8s' cloud (`juju show-controller my-first-juju-controller`), and holding
+  1. a Juju model (i.e., workspace; corresponding to a Kubernetes namespace) implicitly called 'controller', using resources from the 'microk8s' cloud (`juju switch controller`, then `juju show-model controller`), and  holding
+  1. a Juju application (i.e., deployed charm -- in this case, the `juju-controller` charm) called 'controller' with a Juju unit (i.e., replica; corresponding to a Kubernetes pod) called 'controller/0', deployed on a resource from the 'microk8s' cloud (`juju status`), and holding
+  1. the same 3 containers we saw on the MicroK8s side (`juju ssh --container [api-server | charm | mongodb] 0`), which hold the various bits of a running Juju controller:
+    - 'charm': contains a Juju unit agent and `juju-controller` charm code. The Juju unit agent pings the Juju controler agent (see next item) to check the unit's target state against the state in the database (see two items down) and progresses the unit's state by executing the charm code.
```

Comment:
```suggestion
    - 'charm': contains a Juju unit agent and `juju-controller` charm code. The Juju unit agent sends requests to the Juju controller agent (see next item) to check the unit's target state against the state in the database (see two items down) and progresses the unit's state by executing the charm code.
```

Ping means something different, and I don't want to mix terminology here.

Reply:
Updated, thanks!